providers/common/bio_prov.c

```diff
@@ -7,12 +7,15 @@
  * https://www.openssl.org/source/license.html
  */
 
+#include <assert.h>
 #include <openssl/core_numbers.h>
+#include "internal/cryptlib.h"
 #include "prov/bio.h"
 
 static OSSL_BIO_new_file_fn *c_bio_new_file = NULL;
 static OSSL_BIO_new_membuf_fn *c_bio_new_membuf = NULL;
 static OSSL_BIO_read_ex_fn *c_bio_read_ex = NULL;
+static OSSL_BIO_write_ex_fn *c_bio_write_ex = NULL;
```

Comment:
Why is this needed?
The initial reason for adding the bio calls was for fips verification.

Reply:
Its used by the serializers.

Reply:
explicit initialization to null/zero/0.0/etc for globals, yuk.  I think the project should come up with a style policy on this.